third_party/README.md

```diff
@@ -0,0 +1,8 @@
+# third_party
+This directory copies("vendors") files from the following repositories, as required for compiling proto files.
+We periodically sync the local copy with upstream.
+
+1.Repo
+https://github.com/googleapis/googleapis/
+
+files: [google/api/annotations.proto](https://github.com/googleapis/googleapis/blob/c979fbeb80d1b940790a69183629d799e4c05b4c/google/api/annotations.proto) and [google/api/http.proto](https://github.com/googleapis/googleapis/blob/c979fbeb80d1b940790a69183629d799e4c05b4c/google/api/http.proto)
```

Comment:
Could we explicitly state that we should also update the hyperlinks here when we sync the local copy with upstream?